js/apps/account-ui/src/account-security/SigningIn.tsx

```diff
@@ -113,115 +113,108 @@ const SigningIn = () => {
-                                      userLabel: label(meta.credential),
-                                      error,
-                                    }).toString(),
-                                  );
-                                }
-                              }}
-                            />
-                          ) : (
-                            <Button
-                              variant="secondary"
-                              onClick={() => {
-                                if (container.updateAction)
-                                  login({ action: container.updateAction });
-                              }}
-                            >
-                              {t("update")}
-                            </Button>
-                          )}
-                        </DataListAction>,
-                      ]}
-                    />
-                  </DataListItemRow>
-                </DataListItem>
-              ))}
-            </DataList>
-          </PageSection>
-        ))}
-      </DataList>
+            {container.userCredentialMetadatas.map((meta) => (
+              <DataListItem key={meta.credential.id}>
+                <DataListItemRow>
+                  <DataListItemCells
+                    className="pf-u-py-0"
+                    dataListCells={[
+                      ...credentialRowCells(meta),
+                      <DataListAction
+                        key="action"
+                        id={`action-${meta.credential.id}`}
+                        aria-label={t("updateCredAriaLabel")}
+                        aria-labelledby={`cred-${meta.credential.id}`}
+                      >
+                        {container.removeable ? (
+                          <ContinueCancelModal
+                            buttonTitle="remove"
+                            buttonVariant="danger"
+                            modalTitle={t("removeCred", [
+                              label(meta.credential),
+                            ])}
+                            onContinue={async () => {
+                              try {
+                                await deleteCredentials(meta.credential);
+                                addAlert(
+                                  t("successRemovedMessage", {
+                                    userLabel: label(meta.credential),
+                                  }),
+                                );
+                                refresh();
+                              } catch (error) {
+                                addError(
+                                  t("errorRemovedMessage", {
+                                    userLabel: label(meta.credential),
+                                    error,
+                                  }).toString(),
+                                );
+                              }
+                            }}
+                          ></ContinueCancelModal>
```

Comment:
This can be self closing, as the original code.
```suggestion
                          />
```